src/internal_events/common.rs

```diff
@@ -86,12 +86,33 @@ impl InternalEvent for AwsBytesSent {
     fn emit_metrics(&self) {
         counter!(
             "component_sent_bytes_total", self.byte_size as u64,
-            "protocol" => "https",
+            "protocol" => "http(s)",
```

Comment:
I might just leave this as `https` for now. We can do a sweep later to fix this everywhere.

Reply:
oops, I meant to leave it as-is. Just switched it to this to see what it looked like